llvm/tools/llvm-cgdata/Opts.td

```diff
@@ -0,0 +1,28 @@
+include "llvm/Option/OptParser.td"
+
+class F<string letter, string help> : Flag<["-"], letter>, HelpText<help>;
```

Comment:
User-facing tools typically use `--foo` (two dashes) style long options. Consider switching to it?

Reply:
Updated by using two dashes for the long options while using single dash for the short (alias) option.